src/components/storagePools/storagePoolDelete.jsx

```diff
@@ -207,10 +212,11 @@ export class StoragePoolDelete extends React.Component {
 StoragePoolDelete.propTypes = {
     storagePool: PropTypes.object.isRequired,
     vms: PropTypes.array.isRequired,
-    close: PropTypes.func.isRequired,
 };
 
 export const StoragePoolDeleteAction = ({ storagePool, open, vms }) => {
```

Comment:
This `open` parameter is not used anywhere in this function anymore.

Reply:
Good catch, thanks!  Fixed